RadixWallet/Features/SettingsFeature/AccountSecurity/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhrase+View.swift

```diff
@@ -105,17 +99,17 @@ private extension View {
 private extension ManualAccountRecoverySeedPhrase.View {
 	private func subtitle(isOlympia: Bool) -> String {
 		if isOlympia {
-			"Choose the Olympia seed phrase to use for derivation" // FIXME: Strings
+			"Choose the \"Legacy\" Olympia seed phrase for use for derivation:" // FIXME: Strings
 		} else {
-			"Choose the Babylon seed phrase to use for derivation" // FIXME: Strings
+			"Choose the Babylon seed phrase for use for derivation:" // FIXME: Strings
```

Comment:
this sounds wrong "phrase for use for derivation:". hmm better with "phrase for derivation with:" - check with Matt :)